projects/demo/src/modules/tables/table-filters/examples/import/insert-template.md

```diff
@@ -0,0 +1,49 @@
+```html
+<table
+  tuiTableFilters
+  tuiTable
+  [columns]="columns"
+>
+  <thead tuiThead>
+    <tr tuiThGroup>
+      <th
+        *tuiHead="'name'"
+        tuiTh
+      >
+        Name
+      </th>
+      <th
+        *tuiHead="'balance'"
+        tuiTh
+      >
+        <tui-input-number
+          tuiTableFilter="balance"
```

Comment:
VERY small nit:

We can inject `TuiHeadDirective` (with `@Optional()`) inside `tuiTableFilter` and get its name.
It can allow to write in this way)) 
```html
 <th
    *tuiHead="'balance'"
    tuiTh
>
    <tui-input-number
        tuiTableFilter
        [tuiGenericFilter]="filter"
        [formControl]="balance"
    >
          Minimal balance
    </tui-input-number>
</th>
```